ref/Image_Manipulation_and_Rendering.md

```diff
@@ -314,6 +312,35 @@ It plots a point with the corresponding color and opacity and by this creates a
 
 ------
 
+#### Getting pixel data: `imagergb(‹imagename›,‹int›,‹int›)`
+
+**Description:**
+This function does the same as `imagergba(‹imagename›,‹int›,‹int›)`. In particular, it also delivers the alpha value in order to preserve backward compatibility.
+
```

Comment:
I think, this function only exists because of backward compatibility reasons. Most users should use imagergb$4